resources/assets/sass/app.scss

```diff
@@ -15,3 +15,13 @@ $forge-path: "/assets/vendor/forge/assets/";
   padding: 16px;
 }
 
+.button {
+  &.-danger {
+    background-color: #ff4747;
```

Comment:
We should be able to use `$error-color` here since we are pulling in the forge assets. just to keep things consistent